core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -228,6 +259,80 @@ export class DragDropService {
         y: ui.offset.top
       }
     });
+
+    if (this.suggestionOperator !== undefined) {
+      const operatorID = this.suggestionOperator.operatorID;
+      const status = true;
+      this.operatorSuggestionUnhighlightStream.next({status, operatorID});
+      this.suggestionOperator = undefined;
+    }
+  }
+
+  private handleMouseMove(): void {
+    let isDone = false;
```

Comment:
change this variable to `isOperatorDropped`

Reply:
Fixed it.